config/config.go

```diff
@@ -397,6 +397,9 @@ func NewConfig() (c *Config) {
 				WhiteListIstioSystem: []string{"jaeger-query", "istio-ingressgateway"},
 			},
 		},
+		HealthConfig: HealthConfig{
+			Rate: []Rate{},
```

Comment:
I think you should put good defaults here that ensure Kiali behaves the same as today without any configuration ? ie. degraded threshold > 0.1% , failure > 20% ?

Reply:
I'm keeping the defaults in the UI https://github.com/kiali/kiali-ui/pull/1831/files#diff-0e0ca9c6860e3c885bf7cb495a0ad59bR89 . So the proposal is move this default to the config?. My doubt is if user set an empty config we won't have health in the UI

Reply:
I think the default has to be duplicated in UI *and* in backend. The reason is, they have different purposes:
- in the UI, the default is used just as temporary data until server config is fetched; after it's fetched, it has strictly no effect
- in the backend, the default is used to provide data when user hasn't configured anything

Reply:
PS just to let you know: I've re-tested your PR, and the health isn't working because of that, defaults are missing.

Reply:
Added defaults